Firmware/lcd.cpp

```diff
@@ -348,22 +362,56 @@ static uint8_t __attribute__((noinline)) lcd_get_row_offset(uint8_t row)
 void lcd_set_cursor(uint8_t col, uint8_t row)
 {
 	lcd_set_current_row(row);
-	lcd_command(LCD_SETDDRAMADDR | (col + lcd_get_row_offset(lcd_currline)));
+    uint8_t addr = col + lcd_get_row_offset(lcd_currline);
+	lcd_ddram_address = addr;
+	lcd_command(LCD_SETDDRAMADDR | addr);
```

Comment:
Do we need `uint8_t addr`?

I wonder if this saves any flash?
```c
        lcd_ddram_address = col + lcd_get_row_offset(lcd_currline);
	lcd_command(LCD_SETDDRAMADDR | lcd_ddram_address);
```

Reply:
This accesses a global variable. So The code would end up storing to ram, then reading back from ram, which is wasteful. If you use the local variable, then only the store operation to ram will happen and then the cached values in registers will be passed to the functions.

Reply:
Ok if current change is faster, then that's good 👍 I haven't looked at the assembly output.